tools/hyperref-check.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# Extracting all broken hyperref detected by PDFTex
+for file in "./main/acle.md" "./morello/morello.md" "./mve_intrinsics/mve.md" "./neon_intrinsics/advsimd.md"; do
+  echo "Checking $file..."
+
+  broken_refs=`pandoc $file --verbose --fail-if-warnings -o pdfs/tmp.pdf 2>&1 | \
+               grep -E 'pdfTeX warning \(dest\): name{[^}]+}' | sed -E  's/.*name\{([^}]+)\}.*/\1/' | \
+               sort | uniq | tr " " "\n"`
+  number_of_broken_refs=`echo "$broken_refs" | wc -l`
+
+  if [[ $number_of_broken_refs -gt 0 ]]; then
+      echo "**** WARNING! These section links have been found not to be working: "
```

Comment:
It would be good for the warning message to explicitly name the file that is failing:

```suggestion
      echo "**** ERROR! List of unresolved internal references in $file: "
```